package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Fastify compression utils",
   "main": "index.js",
   "dependencies": {
+    "@types/node": "^11.13.9",
```

Comment:
@mbalabash can you move this to devDeps. Thanks

Reply:
Sure

Reply:
Done